include/geos/gconst.h

```diff
@@ -4,14 +4,14 @@
   reassembled by Maciej 'YTM/Elysium' Witkowiak
 */
 
-/* Here are constants which didn't fit into any other cathegory... */
+/* Here are constants which didn't fit into any other category... */
 
 #ifndef _GCONST_H
 #define _GCONST_H
 
-#define NULL            0
-#define FALSE           NULL
+#define NULL            ((void *) 0)
 #define TRUE            0xff
+#define FALSE           ~TRUE
```

Comment:
Please leave the definition of FALSE alone here. This is an entire different can of worms

Reply:
I don't want FALSE=((void *) 0)

Reply:
As said, leave it alone for this PR. defining FALSE as ~TRUE is equally broken in the first place. Someone needs to find out why those are defined in this file at all, and whether they are GEOS specific values.

Reply:
Oh and if (and only if) the tests still happen to fail because of the definition of FALSE, it should be defined as 0 (as it was before)